packages/request-client.js/src/api/request-network.ts

```diff
@@ -190,7 +190,10 @@ export default class RequestNetwork {
     updatedBetween?: Types.ITimestampBoundaries,
     options?: { disablePaymentDetection: boolean },
   ): Promise<Request[]> {
-    if (identity.type !== IdentityTypes.TYPE.ETHEREUM_ADDRESS) {
+    if (
+      identity.type !== IdentityTypes.TYPE.ETHEREUM_ADDRESS &&
```

Comment:
this would require less maintenance in the future if we check a list of supported identities with array.includes